src/main/java/uk/gov/hmcts/reform/blobrouter/services/NewEnvelopesFinder.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.blobrouter.services;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+
+import java.time.Instant;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static java.time.temporal.ChronoUnit.HOURS;
+import static java.util.Collections.singleton;
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class NewEnvelopesFinder {
+
+    private static final Logger logger = getLogger(NewEnvelopesFinder.class);
+
+    private final EnvelopeService envelopeService;
+
+    private final ServiceConfiguration serviceConfig;
+
+    private static final String CRIME_CONTAINER = "crime";
+
+    public NewEnvelopesFinder(
+        EnvelopeService envelopeService,
+        ServiceConfiguration serviceConfig
+    ) {
+        this.envelopeService = envelopeService;
+        this.serviceConfig = serviceConfig;
+    }
+
+    public void checkNewCftEnvelopesCreated() {
+        checkNewEnvelopesInContainers(getCftContainers());
+    }
+
+    public void checkNewCrimeEnvelopesCreated() {
+        checkNewEnvelopesInContainers(singleton(CRIME_CONTAINER));
+    }
+
+    private void checkNewEnvelopesInContainers(Set<String> singleton) {
```

Comment:
weird param name

Reply:
my bad, I refactored and forgot to renamed. its renamed now